mlflow/models/rag_signatures.py

```diff
@@ -10,38 +10,38 @@
     Property,
     Schema,
 )
-from mlflow.utils.annotations import experimental
+from mlflow.utils.annotations import deprecated
 
 
+@deprecated("mlflow.types.llm.ChatMessage")
```

Comment:
@BenWilson2 just wanna confirm that your PR https://github.com/mlflow/mlflow/pull/13712 fixes the deprecation issue we ran into last time

@stevenchen-db would it be possible to call `convert_dataclass_to_schema` on these somewhere in the tests to confirm that the deprecation doesn't end up breaking things on the agents.deploy side of things?